docs/internships/ideas.md

```diff
@@ -244,19 +226,52 @@ Please read the `Research` section of this document for background information
 1. **Difficulty:** Medium
 1. **Impact Definition:** Risky/Exploratory
 
-### Mobile App: Functionality to Allow External Links to be Used Within the Talawa Application
+### Mobile App and API: Ability to data outside/inside Talawa(Posts/Images,Messages) to/from other applications & Link preview machanism in Talawa
```

Comment:
Please make the title shorter.
1. Remove  `and API: Ability` and replace with a colon. It's in the App section of the page. When people read the description they will realize that there are API knowledge requirements.
2. We need a much, much  shorter title so that it is easy to read in the GSoC portal.